src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultApplicationModelProvider.cs

```diff
@@ -224,10 +229,15 @@ protected virtual PropertyModel CreatePropertyModel(PropertyInfo propertyInfo)
             if (bindingInfo == null)
             {
                 var declaringType = propertyInfo.DeclaringType;
-                if (declaringType.IsDefined(typeof(BindPropertyAttribute), inherit: true))
+                var bindPropertyAttribute = declaringType.GetCustomAttribute<BindPropertiesAttribute>(inherit: true);
```

Comment:
`bindPropertiesAttribute`

Reply:
Suggest adding doc comments (somewhere) making it clear custom `IRequestPredicateProvider` attributes applied to a type have precedence over `BindPropertiesAttribute.SupportsGet`.